src/Gui/OverlayWidgets.cpp

```diff
@@ -923,7 +929,7 @@ void OverlayTabWidget::onAction(QAction *action)
     OverlayManager::instance()->refresh(this);
 }
 
-void OverlayTabWidget::setState(State state)
+void OverlayTabWidget::setState(State state, bool user_triggered)

```

Comment:
Every other variable in FreeCAD uses `camelCase` - why this is in the `snake_case`?